core/http/src/main/java/org/trellisldp/http/MultipartUploader.java

```diff
@@ -283,7 +283,8 @@ public Response createBinary(@PathParam("id") final String id, @Context final Co
                             upload.getBinary().getIdentifier(), DC.extent,
                             rdf.createLiteral(size.toString(), XSD.long_))));
 
-            if (resourceService.create(identifier, session, NonRDFSource, dataset.asDataset()).get()) {
+            final IRI container = resourceService.getContainer(rdf.createIRI(TRELLIS_DATA_PREFIX + id)).orElse(null);
+            if (resourceService.create(identifier, session, NonRDFSource, container  , dataset.asDataset()).get()) {
```

Comment:
minor: the `container    ,` might not play well with checkstyle.

Reply:
I amended the checkstyle config to catch these: 474df51a0661f0eb70c5d7904a93bbf9a1bfe72d